cMake/FindSMESH.cmake

```diff
@@ -5,28 +5,55 @@
 # SMESH_INCLUDE_DIR   - where the Salome SMESH include directory can be found
 # SMESH_LIBRARIES     - Link this to use Salome SMESH
 #
- 
 
+# SMESH needs VTK
+find_package(VTK REQUIRED)
+
+# If this definition is not set, linker errors will occur against SMESH on 64 bit machines.
+if(CMAKE_SIZEOF_VOID_P STREQUAL 8)
+	add_definitions(-DSALOME_USE_64BIT_IDS)
```

Comment:
@looooo didn't you run into linker errors with your SMESH build from trelau's repo when this define was set?

Reply:
Yes, true. I solved it by commenting out this line in the smesh recipe :)